src/frontend/citizen-portal/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "yarn run configure -- --environment=dev && ng serve --proxy-config proxy.conf.json",
     "build-dev": "yarn run configure -- --environment=dev && ng build --source-map=false",
     "build-prod": "yarn run configure -- --environment=prod && ng build --configuration production --source-map=false",
+    "api:generate": "docker pull openapitools/openapi-generator-cli:latest && docker run --rm -v \"$(pwd)\"/src/app/api:/local openapitools/openapi-generator-cli generate -i /local/client-api.json -g typescript-angular -t /local/templates --additional-properties=modelFileSuffix=.model -o /local",
```

Comment:
:) I actually meant to install "@openapitools/openapi-generator-cli" as a devDependency so you can run openapi directly and not use docker at all.

As in these examples:
https://github.com/SierraSystems/aer-prototype/blob/main/src/react-frontend/package.json#L61
https://github.com/bcgov/embc-eaf/blob/develop/expense-authorization/src/UI/angular-eteam/package.json#L5

Reply:
ok, I will change it in other branch